# worktree bindernews at a6a8c6edf (HEAD); uncommitted diff saved before removal 2026-10-16
diff --git a/IPlug/IPlugLogger.h b/IPlug/IPlugLogger.h
index f69520a6f..33b0d9fd1 100644
--- a/IPlug/IPlugLogger.h
+++ b/IPlug/IPlugLogger.h
@@ -144,7 +144,7 @@ BEGIN_IPLUG_NAMESPACE
     int i = (int) strlen(cStr);
     cStr[i++] = tz >= 0 ? '+' : '-';
     if (tz < 0) tz = -tz;
-    snprintf(&cStr[i], 32, "%02d%02d", tz / 60, tz % 60);
+    snprintf(&cStr[i], sizeof(cStr) - i, "%02d%02d", tz / 60, tz % 60);
     
     static char sTimeStr[32];
     strcpy(sTimeStr, cStr);
diff --git a/IPlug/IPlugTaskThread.cpp b/IPlug/IPlugTaskThread.cpp
index f9daea8c2..de77a9765 100644
--- a/IPlug/IPlugTaskThread.cpp
+++ b/IPlug/IPlugTaskThread.cpp
@@ -1,4 +1,5 @@
 #include "IPlugTaskThread.h"
+#include <cstdio>
 
 #include <atomic>
 #include <algorithm>
